apidump/apidump.go

```diff
@@ -175,6 +176,23 @@ func Run(args Args) error {
 		traceTags[tags.XAkitaDumpFilterFlag] = args.Filter
 	}
 
+	// Import information about production or staging environment
+	// FIXME: this is performed in apispec as well; does that cause problems if the same
+	// args.Tags map is re-used?
+	{
+		deploymentType, deploymentTags := deployment.GetDeploymentInfo()
+
+		// Override user type, but not CI.
+		if deploymentType != deployment.None &&
+			args.Tags[tags.XAkitaSource] == tags.UserSource || args.Tags[tags.XAkitaSource] == tags.Source("") {
```

Comment:
I think you want parens around the `||` sub-expression: `&&` has higher precedence.

Reply:
Yup, thanks.